client/src/components/base/Preview.tsx

```diff
@@ -1,75 +1,68 @@
 import { useEffect, useState } from "react";
-import { shallowEqual, useDispatch, useSelector } from "react-redux";
 import { useNavigate } from "react-router-dom";
-import { publishGrant, resetStatus } from "../../actions/newGrant";
-import { RootState } from "../../reducers";
+import { useNetwork, useSigner } from "wagmi";
+import { publishGrant } from "../../actions/newGrant";
 import { Status } from "../../reducers/newGrant";
-import { formatDate } from "../../utils/components";
 import { slugs } from "../../routes";
+import { FormInputs, ProjectFormStatus } from "../../types";
+import { formatDate } from "../../utils/components";
 import Details from "../grants/Details";
 import Button, { ButtonVariants } from "./Button";
 import Toast from "./Toast";
 import TXLoading from "./TXLoading";
-import { ProjectFormStatus } from "../../types";
-import { formReset } from "../../actions/projectForm";
```

Comment:
Should we delete `actions/projectForm` and `reducers/projectForm` files if it is no longer being used?

Reply:
Picking this up